app/models/form/sales/pages/nationality1.rb

```diff
@@ -13,6 +13,13 @@ def initialize(id, hsh, subsection)
   end
 
   def questions
-    @questions ||= [Form::Sales::Questions::Nationality1.new(nil, nil, self)]
+    @questions ||= if form.start_year_after_2024?
```

Comment:
can we take this opportunity to rename this page (and associated questions etc) buyer1_nationality for clarity?